### DISCUSSION TITLE
POI announcement missing

### DISCUSSION THREAD

**Discussion by kosi2711** [osmandapp/OsmAnd#11397] [Q&A]
Can anybody confirm that the announcement of POIs during following a route doesn't work anymore? This is at least for Germany (Baden-Württemberg).

I'm using Android 10 on a Motorola G8 Power (not rooted). Same behaviour is on an old LG G4 with LineageOS 16 (rooted). On both devices there is a fresh install of the Fdroid-Version (3.9.10) with the profiles imported from the before used Play-Store-Version which i removed because of the unreliability of GPX altitude measurement. Latest maps are used.

The behaviour with the Play-Store-Version has been the same.

**Comment by Lee-Carre** [comment]
This might be related to #12672.

When using v3(.9.10) I recall not hearing PoI announcements.

However, recently (using v4) I've definitely heard them.
